Sources/FunctionalTableData/CollectionView/FunctionalCollectionData+UIContextMenuInteractionDelegate.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FunctionalCollectionData+UIContextMenuInteractionDelegate.swift
+//  FunctionalTableData
+//
+//  Created by Drake Morin on 2020-03-09.
+//  Copyright © 2020 Shopify. All rights reserved.
+//
+
+import UIKit
+
+extension FunctionalCollectionData {
+	@available(iOS 13.0, *)
+	class CollectionViewContextMenuDelegate: Delegate {
```

Comment:
This separate delegate is needed because of how Apple has implemented Context Menus. Once a delegate implements the function `collectionView(_: contextMenuConfigurationForItemAt: point:)`, then 3D Touch interactions are automatically disabled. 

This means that if I just added these methods to the existing Delegate, I would irreversibly break every 3D Touch implementation on all devices running iOS 13+. Separating it into its own subclass avoids this entirely. It does require that FunctionalCollectionData be initialized with the correct delegate.

Reply:
After additional exploration, I believe a better course of action is to only support context menus on iOS 13+. This is more in line with Apple's intention to deprecate Peek and Pop. It also negates the need for extra delegates, or modifying the `responds(to:)` function.